### ISSUE TITLE
Seeks from a finished video do not trigger StartedSeeking / FinishedSeeking events

### ISSUE THREAD

**Issue by TheloTheGreat** [RenderHeads/UnityPlugin-AVProVideo#669]
On AVPro 2.0.9 for Windows 10, calling Seek on a video that has finished playing (i.e. mp.Control.IsFinished() returns true) does not trigger StartedSeeking and FinishedSeeking events. But there really is a seek operation done, so those events really should trigger.

To repro, in Demo_MediaPlayer, edit MediaPlayerUI.Update by adding the following:
```
if (Input.GetKeyDown(KeyCode.A)) {
	_mediaPlayer.OpenMedia(MediaPathType.RelativeToProjectFolder, "Assets/StreamingAssets/AVProVideoSamples/BigBuckBunny-360p30-H264.mp4", autoPlay: true);
	_mediaPlayer.Control.SetLooping(false);
}
if (Input.GetKeyDown(KeyCode.S)) {
	_mediaPlayer.Control.Seek(4.0);
}
```

Add this to MediaPlayerUI:
```
public virtual void OnMediaPlayerEvent(MediaPlayer mp, MediaPlayerEvent.EventType type, ErrorCode errorCode) {
	Debug.Log("OnMediaPlayerEvent: " + type);
}
```

Add this to MediaPlayer.Start:
```
if (_mediaPlayer) _mediaPlayer.Events.AddListener(OnMediaPlayerEvent);
```

Then press A and let the video finish naturally. Once it's finished, press S.

Expected result: 
Console shows:
"OnMediaPlayerEvent StartedSeeking"
"OnMediaPlayerEvent FinishedSeeking"

Actual result:
Console does not show these lines.

Notes:
- The StartedSeeking and FinishedSeeking events seem to trigger fine as long as the video has not "finished" yet. Reaching the "finish" is what puts the player in a bad state.
- Calling Stop on a finished video, then seeking, still does not trigger StartedSeeking / FinishedSeeking as expected.
- Calling a _second_ seek operation after the video finished will trigger the events
- For these purposes, calling Rewind counts as calling Seek
- On MacOS Big Sur, calling Seek from the finished state does not seek at all (!) so I cannot test whether this is working.

**Comment by AndrewRH** [comment]
Thanks @TheloTheGreat for reporting these edge cases.  We will be investigating 👍

**Comment by TheloTheGreat** [comment]
Fix confirmed on 2.0.10 for both Mac and Windows, thanks.
